package/windows/run.ps1

```diff
@@ -175,12 +179,16 @@ if (-not $CATTLE_NODE_NAME) {
 $CATTLE_NODE_NAME = $CATTLE_NODE_NAME.ToLower()
 
 # check node address #
+$CATTLE_ADDRESS = get-address -Addr $CATTLE_ADDRESS
 $CATTLE_INTERNAL_ADDRESS = get-address -Addr $CATTLE_INTERNAL_ADDRESS
 if (-not $CATTLE_ADDRESS) {
-    $CATTLE_ADDRESS = $CATTLE_INTERNAL_ADDRESS
+    try {
+        $route = Find-NetRoute -RemoteIPAddress 8.8.8.8 | Select-Object -First 1
+        $CATTLE_ADDRESS = $route.IPAddress
+    } catch {}
 }
 if (-not $CATTLE_ADDRESS) {
-    throw "-address is a required option, exit"
+    throw "cannot gain address, please offer -address option, exit"
```

Comment:
gain and offer aren't quite the correct words to use here. I like the old wording better, except for the ending "exit"
"-address is a required option"
Also, it should match all the other errors we throw above this one, so all those should drop ", exist" as well

Reply:
Updated